cheatsheets/SAML_Security_Cheat_Sheet.md

```diff
@@ -94,19 +94,21 @@ The SAML protocol is rarely the vector of choice, though it's important to have
 - Synchronize to a common Internet timesource
 - Define levels of assurance for identity verification
 - Prefer asymmetric identifiers for identity assertions over personally identifiable information (e.g. SSNs, etc)
-- Sign assertions whenever possible
+- Sign each individual Assertion or the entire Response element
 
-## Last Mile Considerations
+## Service Provider (SP) Considerations
 
 - Validating session state for user
 - Level of granularity in setting authZ context when consuming SAML token (do you use groups, roles, attributes)
-- Validate authorized IDP
-- Validate IDP certificates for expiry against CRL/OCSP
+- Ensure each Assertion or the entire Response element is signed
+- Validate signature (see further hints for secure signature checking [above](#validate-protocol-usage))
```

Comment:
I'm not very happy with the structure of the section "Validate Protocol Usage". In my opinion, it's too broad and too unspecific - at least the first part. So, I'm specifically referencing the signature checking part here (starting with "Further vulnerabilities in SAML implementation...").

In addition, I think the first part of the CS contains many details and people may not read and process everything. So, I think, it is good to list the specifically important topics in the Considerations section, instead of just referencing the sections above.
What do you think?

I'm wondering in general, whether the structure of the document could be improved. E.g. with
- splitting up the "Validate Protocol Usage" section
- starting with an overview section (with the most important topics from the "Considerations" section) and then digging into the details of each topic later.
But this would be considerably bigger changes that may need a lot more work. Therefore, I wanted to start with this small PR which is just about the Considerations section.

Please let me know your thoughts or comments.

Reply:
I agree to what you have mentioned. No one wants to read that much details in a cheat sheet. I'd love it if you could open up an issue, in order to address the issue later on.
As for now, I don't mind you keeping the sentence as it. An addition could be done. How about you create an additional subsection for the recommendations, `The following recommendations were proposed` at this stage, and then reference this one?
The subsection should be a second level `##`. 
And this sheet might as well need a markdown refactor.

Reply:
Please see my new commit with my suggestion. I added a # section instead of a subsection, since I personally don't see a big connection to the "Validate Protocol Usage" section.
Please let me know, if you have additional comments.